plugins/modules/files/ini_file.py

```diff
@@ -213,8 +213,8 @@ def do_ini(module, filename, section=None, option=None, value=None,
                 if state == 'present':
                     # change the existing option line
                     if match_opt(option, line):
-                        if not value and allow_no_value:
-                            newline = '%s\n' % option
+                        if value is None and allow_no_value:
+                            newline = u'%s\n' % option
```

Comment:
```suggestion
                            newline = '%s\n' % option
```
This needs to stay without `u`, otherwise this probably breaks on Python 2.

Reply:
I'll fix that. Should I squash the commits?

Reply:
Thanks. No, should be ok like this.